tests/test_train.py

```diff
@@ -66,61 +68,57 @@ def test_train(dataset_path, trained_model_path):
             |_ Area_5.ply
         |_ val_preds_2/ (after second run)
             |_ Area_5.ply
-        |_ parameters.txt
+        |_ config.yml
         |_ training.txt (must contain as many rows as the epoch number times the step number)
         |_ val_IoUs.txt
 
     """
-    max_epoch = 1
-    epoch_steps = 3
-    validation_size = 1
     # First run
     train.train(
         dataset_path,
+        Path("tests/config_S3DIS.yml"),
         chosen_log=None,
         output_dir=trained_model_path,
-        dataset=dataset_path.name,
-        max_epoch=max_epoch,
-        checkpoint_gap=1,
-        epoch_steps=epoch_steps,
-        validation_size=validation_size,
     )
 
     log_dirs = list(trained_model_path.iterdir())
     assert len(log_dirs) == 1
     log_dir = log_dirs[0]
-    assert (log_dir / "parameters.txt").exists()
+
+    config = load_config(Path("tests/config_S3DIS.yml"))
+    assert (log_dir / "config.yml").exists()
```

Comment:
Is `log_dir / "config.yml"` supposed to be equivalent to `tests/config_S3DIS.yml`? We should test that too.

Reply:
Yes it is. I added a test.